### ISSUE TITLE
Code gen

### ISSUE THREAD

**Issue by TheSnowGuru** [YangLing0818/buffer-of-thought-llm#4]
I wonder how this performs on code gen

**Comment by jonathanpv** [comment]
I agree, needs to crawl all previous PRs and create a buffer of thoughts on past solved issues / problems perhaps suggesting which code files to look into or search for.

I prefer vector db of solutions rather than buffer of thoughts but whatever.

Needs to just consume an entire .git repo hope they can add that integration and compare and bench on swe-bench
